Pipeline failuer

Pipeline is failing 

Run npm run tslint
npm ERR! Missing script: "tslint"
npm ERR! 
npm ERR! Did you mean this?
npm ERR!     npm run lint # run the "lint" package script
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2023-08-06T00_[4](https://github.com/Ninad89/ts-template/actions/runs/5773625158/job/15649779181#step:5:5)0_41_32[6](https://github.com/Ninad89/ts-template/actions/runs/5773625158/job/15649779181#step:5:7)Z-debug-0.log
Error: Process completed with exit code 1.

--- a/.github/workflows/node.js.yml
+++ b/.github/workflows/node.js.yml
@@ -25,7 +25,7 @@ jobs:
     - name: Npm install
       run: npm ci
     - name: Lint
-      run: npm run tslint
+      run: npm run lint
     - name: Unit test
       run: npm test
     - name: Build

--- a/README.md
+++ b/README.md
@@ -11,6 +11,7 @@ This project includes
 - eslint setup
 - pretier setup
 - commitlint setup
+- Github actions for CI and npm publish
 ---
 ### How to
 - Get started
@@ -34,6 +35,6 @@ This project includes
 
 ### Todo
  - *How To* guide
- - github actions
+ -  github actions
  - fork this for api with docker setup
  ---

